README.md

```diff
@@ -1,2 +1,47 @@
-# veraison
-Project Veraison will build software components that can be used to build Attestation Verification Services
+<!--
+  -- Copyright 2020 Contributors to the Veraison project.
+  -- SPDX-License-Identifier: Apache-2.0
+--->
+
+<
+# Welcome To VERAISON
+<font size="6
+">
+Project _VERAISON_: **VER**ific**A**t**I**on of atte**S**tati**ON**
+</font>
+
+<p/>
+<p align="center">
+  <img src="./veraison-logo.png"/>
+</p>
+
+## Briefly...
+Project Veraison will build software components that can be used to build Attestation Verification Services.
```

Comment:
Is it intentional to have "s" in Services or it is "software components that can be used to build an Attestation Verification Service"

Reply:
Services plural was intentional, echoing the multiplicity of services outlined later in the doc, however given that this sentence leads that, I can see it might distract, will reword.